client/extensions/woocommerce/state/sites/promotions/README.md

```diff
@@ -0,0 +1,122 @@
+Promotions
+==========
+
+Promotions are a synthetic state construct that are actually created from one of two API objects: coupons, or products (with sale prices). Due to this, a promotion can have one of the following types:
+
+* `product_sale` (for a product with a sale price)
+* `percent` (coupon, matches coupon type)
+* `fixed_cart` (coupon, matches coupon type)
+* `fixed_product` (coupon, matches coupon type)
+
+## Promotions Object
+
+As Promotions only exist in memory state on the client at this point, the definition of a promotion is as follows:
+
+```js
+{
+	// Required properties
+	id: string,
+	name: string,
+	type: string (one of the types above),
+	appliesTo: object,
+
+	// Amount properties (optional),
+	fixedDiscount: number,
+	percentDiscount: number,
+	salePrice: number,
+
+	// Constraint properties (optional)
+	couponCode: string,
+	startDate: date,
+	endDate: date,
+	individualUse: boolean,
```

Comment:
Most of these are self-explanatory, but a few like individualUse or freeShipping might warrant a comment

Reply:
I added comments after each field for clarity, thanks!